cmd/login/login.go

```diff
@@ -68,16 +69,17 @@ func run(client login.Client) func(*cobra.Command, []string) error {
 			return err
 		}
 
+		fullURL := fmt.Sprintf("%s/%s", viper.GetString(cliflags.BaseURIFlag), deviceAuthorization.VerificationURI)
 		var b strings.Builder
 		b.WriteString(fmt.Sprintf("Your code is %s\n", deviceAuthorization.UserCode))
 		b.WriteString("This code verifies your authentication with LaunchDarkly.\n")
-		b.WriteString(
-			fmt.Sprintf(
-				"Press Enter to open the browser or visit %s/%s (^C to quit)\n",
-				viper.GetString(cliflags.BaseURIFlag),
-				deviceAuthorization.VerificationURI,
-			),
-		)
+
+		err = browser.OpenURL(fullURL)
```

Comment:
We may want to revisit this flow to have the user press enter to open the browser instead of opening it automatically. That would be more complex, so we could iterate to it.

Reply:
also, you might consider unconditionally writing "opening browser to https://..." to stdout. 99% of the time the browser will open and the user won't care about the extra info. in the 1% failure, they have a breadcrumb to move forward themselves. up to you.

Reply:
Updated language.